CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ CLI command and its behaviour. There are no guarantees of stability for the
   - Qt .pro (`.pro`) (#632)
   - Textile (`.textile`) (#712)
   - Visual Studio Code workspace (`.code-workspace`) (#746)
+  - Application Resource Bundle (`.arb`) (#748)
```

Comment:
Can we use the Pull Request number here please, so #749?

Reply:
Done. 

I have also modified the previous line, because there were the same problem in my previous pull request.